specs/core/0_beacon-chain.md

```diff
@@ -43,11 +43,11 @@
             - [`ProposalSignedData`](#proposalsigneddata)
         - [Beacon chain state](#beacon-chain-state)
             - [`BeaconState`](#beaconstate)
-            - [`ValidatorRecord`](#validatorrecord)
-            - [`CrosslinkRecord`](#crosslinkrecord)
+            - [`Validator`](#Validator)
+            - [`Crosslink`](#Crosslink)
             - [`ShardCommittee`](#shardcommittee)
             - [`DepositRootVote`](#depositrootvote)
-            - [`PendingAttestationRecord`](#pendingattestationrecord)
+            - [`PendingAttestation`](#PendingAttestation)
```

Comment:
Should be `#pendingattestation` (no capitals)

Reply:
Thanks for a correction.

Reply:
Could you explain to write no capital? it works following header with A capital.
Anyway, Thank you for review and Point my errors.

Reply:
For consistency, I guess. Putting capitals everywhere works also.

Reply:
The generator that @hwwhww uses defaults to lower case. ++ for consistency.

Reply:
Changed all lower case, I will try to check consistency next PR.

Reply:
Oh, which generator? I want :)

Reply:
It's a vscode plugin: https://github.com/AlanWalk/markdown-toc
but I don't use vscode..